### PR TITLE
chore: Fix auto-release configuration

### PR DESCRIPTION
**Pull request by lukelooker**
- Clean-up CHANGELOG for auto-updates
- Fix NPM_TOKEN
- 
### :white_check_mark: Requirements

- [ ] Includes test coverage for all changes
- [ ] Build and tests are passing
- [ ] Update documentation
- [ ] Updated CHANGELOG
- [ ] Checked for i18n impacts
- [ ] Checked for a11y impacts
- [ ] Check for image-snapshot changes (run `yarn image-snapshots` locally)
- [ ] PR is ideally < ~400LOC

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+### Requirements
+
+Please check the following items are addressed in your pull request (or are not applicable)
+
+- [ ] Includes test coverage for all changes
+- [ ] Documentation updated
+- [ ] i18n impacts
+- [ ] a11y impacts
+- [ ] :rotating_light: Check for image-snapshot changes (run `yarn image-snapshots` locally)
```
Comment (lukelooker) [inline .github/pull_request_template.md:15]: Moved this into `.github` folder and removed notes about CHANGELOG (among other minor tweaks)

File: .github/workflows/canary-release.yml [hunk 2]
```diff
@@ -44,4 +44,4 @@ jobs:
       - name: Publish to NPM registry with canary tag
         run: npx lerna publish from-package --yes --dist-tag canary
         env:
-          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```
Comment (lukelooker) [inline .github/workflows/canary-release.yml:47]: Hoping this fixes things.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,18 +1,7 @@
...
-
-NOTE: This is the CHANGELOG for the @looker/components MONOREPO. Each package has a package-specific CHANGELOG as well. Review the Github release notes for a roll-up of all CHANGELOG changes across the monorepo.
-
-- [Components](./packages/components/CHANGELOG.md)
-- [Components Providers](./packages/components-providers/CHANGELOG.md)
-- [Components Test Utilities](./packages/components-providers/CHANGELOG.md)
-- [Components Theme Editor](./packages/theme-editor/CHANGELOG.md)
-- [Design Tokens](./packages/design-tokens/CHANGELOG.md)
-- [Icons](./packages/icons/CHANGELOG.md)
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
```
Comment (lukelooker) [inline CHANGELOG.md:4]: Lerna's auto-CHANGELOG feature expects exactly this header to inject in the right location

**Review by mdodgelooker — APPROVED** [review]
LGTM

**Pull request merged**